common/telemetry/attribute.go

```diff
@@ -0,0 +1,69 @@
+package telemetry
```

Comment:
This is an attempt to unify the way we define tags. I'm not certain that it's useful or even a good idea.